docs/getting_started.md

```diff
@@ -24,26 +24,33 @@ section.
 The following dependencies are required to build the Acts core library:
 
 -   A C++17 compatible compiler (recent versions of either gcc and clang should work)
--   [CMake](https://cmake.org) >= 3.11
--   [Boost](http://boost.org) >= 1.69 with `filesystem`, `program_options`, and `unit_test_framework`
--   [Eigen](http://eigen.tuxfamily.org) >= 3.2.9
+-   [CMake](https://cmake.org) >= 3.14
+-   [Boost](http://boost.org) >= 1.71 with `filesystem`, `program_options`, and `unit_test_framework`
+-   [Eigen](http://eigen.tuxfamily.org) >= 3.3.7
 
 The following dependencies are optional and are needed to build additional
-components.
+components:
 
--   [DD4Hep](http://dd4hep.cern.ch) >= 1.10 for the DD4Hep plugin and some examples
--   [Doxygen](http://doxygen.org) >= 1.8.11 for the documentation
+-   [CUDA](https://developer.nvidia.com/cuda-zone) for the CUDA plugin
+-   [DD4Hep](http://dd4hep.cern.ch) >= 1.11 for the DD4Hep plugin and some examples
+-   [Doxygen](http://doxygen.org) >= 1.8.15 for the documentation
 -   [Geant4](http://geant4.org/) for some examples
--   [HepMC](https://gitlab.cern.ch/hepmc/HepMC3) >= 3.1 for some examples
+-   [HepMC](https://gitlab.cern.ch/hepmc/HepMC3) >= 3.2.1 for some examples
 -   [Pythia8](http://home.thep.lu.se/~torbjorn/Pythia.html) for some examples
--   [Intel Threading Building Blocks](https://01.org/tbb) for the examples
--   [ROOT](https://root.cern.ch) >= 6.10 for the TGeo plugin and the examples
+-   [Intel Threading Building Blocks](https://01.org/tbb) >= 2020.1 for the examples
+-   [ONNX](https://onnx.ai) for the ONNX plugin and some examples
+-   [ROOT](https://root.cern.ch) >= 6.20 for the TGeo plugin and the examples
```

Comment:
Any reason for the reordering?

Reply:
This is ordered alphabetical.

Reply:
Then you need to move Pythia8 after ONNX.

Reply:
Yes, I should. Fixed.